sql/derived_tables/po_lines_cost.sql

```diff
@@ -0,0 +1,29 @@
+DROP TABLE IF EXISTS local.po_lines_cost;
+
+CREATE TABLE local.po_lines_cost AS 
+
+/* Create table for cost information on the purchase order line */

```

Comment:
Please use double dashes (`--`) for comments rather than `/*` and `*/`, for maximum compatibility.  I will fix this after merging.